geowebcache/core/src/main/java/org/geowebcache/storage/BlobStoreAggregator.java

```diff
@@ -0,0 +1,260 @@
+package org.geowebcache.storage;
+
+import com.google.common.base.Preconditions;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.geowebcache.GeoWebCacheException;
+import org.geowebcache.GeoWebCacheExtensions;
+import org.geowebcache.config.BlobStoreConfiguration;
+import org.geowebcache.config.BlobStoreInfo;
+import org.geowebcache.config.ServerConfiguration;
+import org.geowebcache.config.meta.ServiceInformation;
+import org.geowebcache.grid.GridSetBroker;
+import org.geowebcache.util.CompositeIterable;
+import org.springframework.util.Assert;
+
+import java.util.*;
+
+public class BlobStoreAggregator {
+
+    private List<BlobStoreConfiguration> configs;
+    private static Log log = LogFactory.getLog(org.geowebcache.storage.BlobStoreAggregator.class);
+    private ServiceInformation serviceInformation;
+    private GridSetBroker gridSetBroker;
+
+
+
+
+    public BlobStoreAggregator(GridSetBroker gridSetBroker, List<BlobStoreConfiguration> configs) {
```

Comment:
As far as I can tell, GridSetBroker is just necessary for TileLayerDispatcher, not required for BlobStoreAggregator.